### DISCUSSION TITLE
Why are some of the ligature and glyph examples red?

### DISCUSSION THREAD

**Discussion by schuelermine** [tonsky/FiraCode#1403] [General]
For instance, in https://github.com/tonsky/FiraCode/raw/master/extras/ligatures.png, https://github.com/tonsky/FiraCode/raw/master/extras/character_variants.png, https://github.com/tonsky/FiraCode/raw/master/extras/ligature_variants.png, https://github.com/tonsky/FiraCode/raw/master/extras/console.png, https://github.com/tonsky/FiraCode/raw/master/extras/math.png

**Comment by tonsky** [comment]
This indicates what was added in the last version
